sdk/servicebus/azure-servicebus/azure/servicebus/common/mixins.py

```diff
@@ -288,6 +288,10 @@ def __init__(self, address, name, shared_access_key_name=None,
             'key_name': shared_access_key_name,
             'shared_access_key': shared_access_key_value}
 
+        transport_type = kwargs.get('transport_type') or None
+        if transport_type is not None:
+            self.auth_config['transport_type'] = transport_type
+
         self.mgmt_client = kwargs.get('mgmt_client') or ServiceBusService(
```

Comment:
Default value should not be None, it should be TransportType.Amqp, the code can be simplified to:
`self.auth_config['transport_type'] = kwargs.get('transport_type') or TransportType.Amqp.`

and add `from uamqp.constants import TransportType` below `from uamqp import Source`